h2o-py/h2o/model/model_base.py

```diff
@@ -529,10 +544,16 @@ def coef_norm(self):
 
         These coefficients can be used to evaluate variable importance.
         """
-        tbl = self._model_json["output"]["coefficients_table"]
-        if tbl is None:
-            return None
-        return {name: coef for name, coef in zip(tbl["names"], tbl["standardized_coefficients"])}
+        if self._model_json["output"]["model_category"]=="Multinomial":
+            tbl = self._model_json["output"]["standardized_coefficient_magnitudes"]
+            if tbl is None:
+                return None
+            return {name: coef for name, coef in zip(tbl["names"], tbl["coefficients"])}
+        else:
+            tbl = self._model_json["output"]["coefficients_table"]
+            if tbl is None:
+                return None
+            return {name: coef for name, coef in zip(tbl["names"], tbl["standardized_coefficients"])}
 
```

Comment:
I cannot combine the two because multinomial coef_norm are always +ve while for other families, they can be either positive or negative